### PR TITLE
7.0 web widget boolean switch

### PR DESCRIPTION
**Pull request by petrus-v**
This module add a widget `boolean_switch` to render boolean fields. One
of it's main features is to quick edit that field without enter in edit mode
from list view or form view.

There is some demo data in this module to let functional people to  test that feature on users menu.

### CODE REVIEW

**Comment by petrus-v** [comment]
I'm waiting #232 before rebasing to avoid pylint complaints.

**Comment by petrus-v** [comment]
Just rebased this branch and let see if Travis is happy!

**Comment by petrus-v** [comment]
rebased because runbot needs #242

**Comment by JonathanNEMRY** [comment]
@petrus-v  Great module!
I've made some functional test on runbot. Looks great but I noticed a strange behavior (tested into the user form): 
- Switch true to false
- Click on edit
- Click on discard
- Previous value comes back

Is there an explaination?

**Comment by petrus-v** [comment]
@JonathanNEMRY,

Thanks for review! well seen. I'm trying to unit test it!

**Comment by legalsylvain** [comment]
Hi @petrus-v ? Did you took the time to check @JonathanNEMRY remarks ?

regards.

**Comment by petrus-v** [comment]
@legalsylvain,

I've reproduce it and started to do some QUnit test, unfortunately I'm busy and we are using this widget on production only on tree view for the moment... that why I'm not hurry about this PR, but it's on my TODO list

Regards

**Comment by petrus-v** [comment]
@legalsylvain Last commit fix issue found by @JonathanNEMRY.

Also I rebased the branch

**Comment by petrus-v** [comment]
Ooops I had pushed on the bad repo!

**Comment by OSguard** [comment]
:+1: testet on runbot

File: web_widget_boolean_switch/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+~~~~~
+``extra`` is used to set
+`bootstrap-switch <http://www.bootstrap-switch.org/options.html>`_ options.
+
+*Available options*:
+
+   * **size**: The checkbox size - default: `null` - values: null, 'mini',
+     'small', 'normal', 'large'
+   * **animate**: Animate the switch - default: `true`
+   * **indeterminate**: Indeterminate state - default: `false`
```
Comment (guewen) [inline web_widget_boolean_switch/__openerp__.py:88]: I'm not sure this option has much sense for Odoo since it has no indeterminate values.

**Comment by guewen** [comment]
Impressive work on the tests! :+1:

File: web_widget_boolean_switch/__openerp__.py (hunk 1 above)
Comment (petrus-v) [inline web_widget_boolean_switch/__openerp__.py:88]: You're right, I move it in the warning section!

Comment (guewen) [inline web_widget_boolean_switch/__openerp__.py:88]: Thanks

**Pull request merged**